src/TroykaTextLCD.h

```diff
@@ -6,96 +6,96 @@
+constexpr uint8_t LCD_DISPLAYOFF = 0x00;
+constexpr uint8_t LCD_CURSORON = 0x02;
+constexpr uint8_t LCD_CURSOROFF = 0x00;
+constexpr uint8_t LCD_BLINKON = 0x01;
+constexpr uint8_t LCD_BLINKOFF = 0x00;
 
 // flags for display/cursor shift (LCD_CURSORSHIFT)
-#define LCD_DISPLAYMOVE 0x08
-#define LCD_CURSORMOVE 0x00
-#define LCD_MOVERIGHT 0x04
-#define LCD_MOVELEFT 0x00
+constexpr uint8_t LCD_DISPLAYMOVE = 0x08;
+constexpr uint8_t LCD_CURSORMOVE = 0x00;
+constexpr uint8_t LCD_MOVERIGHT = 0x04;
+constexpr uint8_t LCD_MOVELEFT = 0x00;
 
 // flags for function set (LCD_FUNCTIONSET)
-#define LCD_8BITMODE 0x10
-#define LCD_4BITMODE 0x00
-#define LCD_2LINE 0x08
-#define LCD_1LINE 0x00
-#define LCD_5x10DOTS 0x04
-#define LCD_5x8DOTS 0x00
+constexpr uint8_t LCD_8BITMODE = 0x10;
+constexpr uint8_t LCD_4BITMODE = 0x00;
+constexpr uint8_t LCD_2LINE = 0x08;
+constexpr uint8_t LCD_1LINE = 0x00;
+constexpr uint8_t LCD_5x10DOTS = 0x04;
+constexpr uint8_t LCD_5x8DOTS = 0x00;
 // Normal or Extension instruction select
-#define LCD_NM_INSTRUCTION 0x00
-#define LCD_EX_INSTRUCTION 0x01
+constexpr uint8_t LCD_NM_INSTRUCTION = 0x00;
+constexpr uint8_t LCD_EX_INSTRUCTION = 0x01;
 
 // bias selection / Internal OSC frequency adjust
-#define LCD_EX_SETBIASOSC 0x10
+constexpr uint8_t LCD_EX_SETBIASOSC = 0x10;
 // set ICON RAM address
-#define LCD_EX_SETICONRAMADDR 0x40
+constexpr uint8_t LCD_EX_SETICONRAMADDR = 0x40;
 // power / ICON control / Contrast set (high byte)
-#define LCD_EX_POWICONCONTRASTH 0x50
+constexpr uint8_t LCD_EX_POWICONCONTRASTH = 0x50;
 // follower control
-#define LCD_EX_FOLLOWERCONTROL 0x60
+constexpr uint8_t LCD_EX_FOLLOWERCONTROL = 0x60;
 // contrast set (low byte)
-#define LCD_EX_CONTRASTSETL 0x70
+constexpr uint8_t LCD_EX_CONTRASTSETL = 0x70;
 
 // flags for Bias selection
 // bias will be 1/4
-#define LCD_BIAS_1_4 0x08
+constexpr uint8_t LCD_BIAS_1_4 = 0x08;
 // bias will be 1/5
-#define LCD_BIAS_1_5 0x00
+constexpr uint8_t LCD_BIAS_1_5 = 0x00;
 
 // flags Power / ICON control / Contrast set(high byte)
 // ICON display on
-#define LCD_ICON_ON 0x08
+constexpr uint8_t LCD_ICON_ON = 0x08;
 // ICON display off
-#define LCD_ICON_OFF 0x00
+constexpr uint8_t LCD_ICON_OFF = 0x00;
 // booster circuit is turn on
-#define LCD_BOOST_ON 0x04
+constexpr uint8_t LCD_BOOST_ON = 0x04;
 // booster circuit is turn off
-#define LCD_BOOST_OFF 0x00
-#define LCD_OSC_122HZ 0x00
-#define LCD_OSC_131HZ 0x01
-#define LCD_OSC_144HZ 0x02
-#define LCD_OSC_161HZ 0x03
-#define LCD_OSC_183HZ 0x04
-#define LCD_OSC_221HZ 0x05
-#define LCD_OSC_274HZ 0x06
-#define LCD_OSC_347HZ 0x07
+constexpr uint8_t LCD_BOOST_OFF = 0x00;
+constexpr uint8_t LCD_OSC_122HZ = 0x00;
+constexpr uint8_t LCD_OSC_131HZ = 0x01;
+constexpr uint8_t LCD_OSC_144HZ = 0x02;
+constexpr uint8_t LCD_OSC_161HZ = 0x03;
+constexpr uint8_t LCD_OSC_183HZ = 0x04;
+constexpr uint8_t LCD_OSC_221HZ = 0x05;
+constexpr uint8_t LCD_OSC_274HZ = 0x06;
+constexpr uint8_t LCD_OSC_347HZ = 0x07;
```

Comment:
IMHO here "enum" will be better, for example:
enum LcdOsc { F122Hz = 0, F131Hz = 1, ... F347Hz = 7 };